base/random.jl

```diff
@@ -257,10 +257,21 @@ globalRNG() = GLOBAL_RNG
 Pick a random element or array of random elements from the set of values specified by `S`; `S` can be
 
 * an indexable collection (for example `1:n` or `['x','y','z']`), or
+* a `Dict`, a `Set` or an `IntSet`, or
 * a type: the set of values to pick from is then equivalent to `typemin(S):typemax(S)` for
   integers (this is not applicable to `BigInt`), and to ``[0, 1)`` for floating point numbers;
 
 `S` defaults to `Float64`.
+
+```julia-repl
+julia> rand(Int, 2)
+2-element Array{Int64,1}:
+ 1339893410598768192
+ 1575814717733606317
+
+julia> rand(MersenneTwister(0), Dict(1=>2, 3=>4))
```

Comment:
Actually, I think I wanted to illustrate different combinations of arguments and show the use of an explicit RNG (and `MersenneTwister` is the most likely to be used, but doesn't accept for now no-argument), so I would rather keep it as is.